src/meta/service/src/cloud_service.rs

```diff
@@ -1,197 +1 @@
-// Copyright 2024 RisingWave Labs
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 
-use std::collections::BTreeMap;
-use std::sync::LazyLock;
-
-use async_trait::async_trait;
-use regex::Regex;
-use risingwave_connector::error::ConnectorResult;
-use risingwave_connector::source::kafka::private_link::insert_privatelink_broker_rewrite_map;
-use risingwave_connector::source::{
-    ConnectorProperties, SourceEnumeratorContext, SourceProperties, SplitEnumerator,
-};
-use risingwave_connector::{dispatch_source_prop, WithOptionsSecResolved};
-use risingwave_meta::manager::MetadataManager;
-use risingwave_meta_model_v2::ConnectionId;
-use risingwave_pb::catalog::connection::Info::PrivateLinkService;
-use risingwave_pb::cloud_service::cloud_service_server::CloudService;
-use risingwave_pb::cloud_service::rw_cloud_validate_source_response::{Error, ErrorType};
-use risingwave_pb::cloud_service::{
-    RwCloudValidateSourceRequest, RwCloudValidateSourceResponse, SourceType,
-};
-use thiserror_ext::AsReport;
-use tonic::{Request, Response, Status};
-
-use crate::rpc::cloud_provider::AwsEc2Client;
-
-pub struct CloudServiceImpl {
-    metadata_manager: MetadataManager,
-    aws_client: Option<AwsEc2Client>,
-}
-
-impl CloudServiceImpl {
-    pub fn new(metadata_manager: MetadataManager, aws_client: Option<AwsEc2Client>) -> Self {
-        Self {
-            metadata_manager,
-            aws_client,
-        }
-    }
-}
-
-#[inline(always)]
-fn new_rwc_validate_fail_response(
-    error_type: ErrorType,
-    error_message: String,
-) -> Response<RwCloudValidateSourceResponse> {
-    Response::new(RwCloudValidateSourceResponse {
-        ok: false,
-        error: Some(Error {
-            error_type: error_type.into(),
-            error_message,
-        }),
-    })
-}
-
-#[async_trait]
-impl CloudService for CloudServiceImpl {
-    async fn rw_cloud_validate_source(
```

Comment:
This rpc interface is provided to the cloud portal previously, I am not sure whether it is still valid. Please confirm with @cloudcarver @neverchanje 
https://github.com/risingwavelabs/risingwave/issues/10139

Reply:
@Gogomoe worked on this previously

Reply:
The cloud service is call via cloudagent rpc, according to https://risingwave-labs.slack.com/archives/C03DFGV6L2W/p1729495045551239?thread_ts=1729479931.025979&cid=C03DFGV6L2W. 
`PingSource` func is used in https://github.com/risingwavelabs/risingwave-cloud/blob/main/mgmt/internal/apiservice/controllers/controller_v1/source.go#L124 and https://github.com/risingwavelabs/risingwave-cloud/blob/main/mgmt/internal/apiservice/controllers/controller_v2/sources.go#L160. 

But upper level func `PostTenantsNsIdSourcesPing` and `PostTenantTenantIdSourcesPing` are not called in the codebase. Seeing them as deprecated. 

Will proceed on removing the service.